virtcontainers/kata_agent.go

```diff
@@ -1769,6 +1798,12 @@ func (k *kataAgent) copyFile(src, dst string) error {
 		Gid:      int32(st.Gid),
 	}
 
+	// Handle the special case where the file is empty
```

Comment:
@sboeuf why was this needed?

Reply:
Because the file was not copied at all when the size was 0, because of the logic below.